src/modules/displayScore.js

```diff
@@ -0,0 +1,16 @@
+function displayScores(data) {
+  data.forEach((set) => {
+    const tbody = document.querySelector('tbody');
+    const tr = document.createElement('tr');
+    tr.classList.add('list-item');
+    const tdName = document.createElement('td');
+    tdName.innerHTML = `${set.user}`;
+    const tdScore = document.createElement('td');
+    tdScore.innerHTML = `${set.score}`;
+    tr.appendChild(tdName);
+    tr.appendChild(tdScore);
+    tbody.appendChild(tr);
+  });
+}
+
+export default displayScores;
```

Comment:
- [ ] Kindly use `arrow functions` instead of the `function` keyword for the `displayScores` function as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20arrow%20functions%20instead%20of%20the%20function%20keyword.).